cpp/include/cudf/wrappers/timestamps.hpp

```diff
@@ -42,33 +40,37 @@ using timestamp = time_point<Duration>;
  */
 
 /**
- * @brief Type alias representing an int32_t duration of days since the unix
- * epoch.
+ * @brief Type alias representing a cudf::duration_D (int32_t) since the unix epoch.
+ */
+using timestamp_D = detail::timestamp<cudf::duration_D>;
```

Comment:
I don't think we need/want to add these type aliases. These are all for our publicly exposed chrono types. We don't expose public types for hours/minutes. Just use the corresponding `cuda::std::chrono` types directly.

Reply:
Just to make sure, we need `duration_m` and `duration_h` exposed to the public but neither `timestamp_h` nor `timestamp_m`? Another reason I'm adding hours/minutes is that both hour timepoint and minute timepoint will be used when refactoring `extract_time` function in `io/csv/datetime.cuh`

Reply:
[Here](https://github.com/rapidsai/cudf/blob/2a34daf04af781a188b4d844a48493ee3723813a/cpp/include/cudf/types.hpp#L203-L234) is the list of our public types. 

I see you're right that we have `duration_m` and `duration_h` aliases even though those don't exist as column types. I guess what you're doing is fine then.